Formula/ignition-plugin2.rb

```diff
@@ -2,13 +2,13 @@ class IgnitionPlugin2 < Formula
   desc "Plugin libraries for robotics applications"
   homepage "https://github.com/ignitionrobotics/ign-plugin"
   url "https://github.com/ignitionrobotics/ign-plugin.git", branch: "main"
-  version "1.999.999~1"
+  version "1.999.999~0~20220406"
```

Comment:
I would have thought this was a version decrease and that the audit script would have complained

Reply:
but I guess it's fine

Reply:
Odd o__o, according to the semver spec it is a version decrease...